### PR TITLE
Update package.json for cpu and webgpu

### PR DESCRIPTION
**Pull request by axinging**
To see the logs from the Cloud Build CI, please join either our [discussion](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs) or [announcement](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs-announce) mailing list.

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/5901)

### CODE REVIEW

**Comment by axinging** [comment]
@haoyunfeix @qjia7  @xhcao @gyagp PTAL

File: tfjs-backend-cpu/yarn.lock [hunk 1]
```diff
@@ -313,6 +324,11 @@
   resolved "https://registry.yarnpkg.com/@types/seedrandom/-/seedrandom-2.4.27.tgz#9db563937dd86915f69092bc43259d2f48578e41"
   integrity sha1-nbVjk33YaRX2kJK8QyWdL0hXjkE=
 
+"@types/webgl-ext@0.0.30":
```
Comment (qjia7) [inline tfjs-backend-cpu/yarn.lock:327]: Don't understand why "@types/webgl-ext@0.0.30" is needed for cpu and webgpu. @mattsoulanille

Comment (mattsoulanille) [reply tfjs-backend-cpu/yarn.lock:327]: `yarn why @types/webgl-ext` says that `@tensorflow/tfjs-core` depends on it. I think this is unintentional. Running `yarn remove @types/webgl-ext` in core and then running `yarn build-deps` and `yarn install` in backend-cpu seems to fix it.

Comment (axinging) [reply tfjs-backend-cpu/yarn.lock:327]: Thanks @mattsoulanille , I will close this change.

**Pull request closed without merging**